mlflow/utils/search_utils.py

```diff
@@ -15,9 +16,9 @@
 
 class SearchUtils(object):
     VALID_METRIC_COMPARATORS = set(['>', '>=', '!=', '=', '<', '<='])
-    VALID_PARAM_COMPARATORS = set(['!=', '='])
-    VALID_TAG_COMPARATORS = set(['!=', '='])
-    VALID_STRING_ATTRIBUTE_COMPARATORS = set(['!=', '='])
+    VALID_PARAM_COMPARATORS = set(['!=', '=', 'LIKE', 'like', 'ILIKE', 'ilike'])
```

Comment:
as long as operators are converted to **UPPER** case, only `LIKE` and `ILIKE` should be required.
Prefer **UPPER** case over lower case to distinguish SQL tokens from python code.

Reply:
Done